html/css/main.css

```
@@ -397,6 +397,16 @@ body {
 	height: 23vh;
 }
 
+.iteminfo-content {
+    width: fit-content;
+    float: right;
```

```suggestion
    float: right;
```
Why is float used? Considering it hasnt needed this previously?